src/agoradatatools/great_expectations/gx/expectations/target_exp_validation_harmonized.json

```diff
@@ -0,0 +1,345 @@
+{
+  "data_asset_type": null,
+  "expectation_suite_name": "target_exp_validation_harmonized",
+  "expectations": [
+    {
+      "expectation_type": "expect_column_values_to_be_of_type",
+      "kwargs": {
+        "column": "ensembl_gene_id",
+        "type_": "str"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_not_be_null",
+      "kwargs": {
+        "column": "ensembl_gene_id"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_value_lengths_to_equal",
+      "kwargs": {
+        "column": "ensembl_gene_id",
+        "value": 15
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_match_regex",
+      "kwargs": {
+        "column": "ensembl_gene_id",
+        "regex": "^ENSG\\d{11}$"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_of_type",
+      "kwargs": {
+        "column": "hgnc_symbol",
+        "type_": "str"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_value_lengths_to_be_between",
+      "kwargs": {
+        "column": "hgnc_symbol",
+        "max_value": 100,
+        "min_value": 2
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_match_regex",
+      "kwargs": {
+        "column": "hgnc_symbol",
+        "regex": "^[a-zA-Z0-9_() ]*$"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_of_type",
+      "kwargs": {
+        "column": "hypothesis_tested",
+        "type_": "str"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_value_lengths_to_be_between",
+      "kwargs": {
+        "column": "hypothesis_tested",
+        "max_value": 1000,
+        "min_value": 2
+      },
+      "meta": {}
+    },
```

Comment:
@JessterB I am looking into this. Do you have an example dataset that has this replacement character in it that I could use for testing?

Reply:
@JessterB I was able to create my own testing dataset and confirmed that 
```
validator.expect_column_values_to_not_match_regex("hypothesis_tested", "\ufffd")
```
will fail when the `hypothesis_tested` column includes `�`/`\ufffd` in its contents. I will implement this for the columns you listed above.

Follow-up question: I currently have expectations
```
validator.expect_column_values_to_be_in_set("species", [
        "Mouse",
        "Human",
        "Human, Mouse",
        "Human, Drosophila",
        "Drosophila",
        "Drosophila, Human",
        "Human ", 
        "Zebrafish",
    ]
)
```
and
```
validator.expect_column_values_to_be_in_set("outcome_measure", [
        "Behavioral",
        "Electrophysiology",
        "Biochemical",
        "Biomarker",
        "Cell Biology",
        "Other",
        "Histopathology",
        "Imaging",
        "Omics",
        "Immunochemistry",
        "Immunohistochemistry",
    ]
)
```
which, if we include would make it so that the `expect_column_values_to_not_match_regex` expectation is not needed for `species` and `outcome_measure`. Which expectation should I include for these fields?

Reply:
Having both seems harmless, but if you want to keep the number of expectations to a minimum I'd prefer to keep the enum expectations, as our contributers are not at all vigilant about consistency.

Reply:
P.S. Thank you for adding the regex for the replacement character! It will be really useful for a number of datasets that contain free text gathered from collaborators via Excel spreadsheets. Someday we'd like to use DFA instead of Excel for ingesting this data, but when we will be able to implement DFA for Agora remains TBD...

Reply:
No problem. I don't see any harm in having both in there, so both it is!